### ISSUE TITLE
Calculate accuracy on both train and test?

### ISSUE THREAD

**Issue by biagio90** [BVLC/caffe#1345]
Hi everybody,
I am working on plotting statistics like the accuracy over iterations. I use parse_log and gnuplot for that. Now my problem is that I need accuracy for both testset and trainset. In other words, I want that the TEST phase will be applied on both databases and as a result give me two accuracy numbers in order to see how the two courves evolve. It should also have different test iteration, since the training set is usually bigger than the test set.

Is it possible somehow?
Thanks

**Comment by yghlc** [comment]
test
